src/test.js

```diff
@@ -238,6 +238,33 @@ describe('autodux().selectors', async should => {
 describe('autodux() action creators', async should => {
   const { assert } = should();
 
+  {
+    const {
+      actions: {
+        setUserName
+      }
+    } = autodux({
+      slice: 'user',
+      initial: {
+        userName: 'Anonymous',
+      }
+    });
+    const userName = 'Foo';
+
+    const actual = setUserName(userName);
+    const expected = {
+      type: 'user/setUserName',
+      payload: userName,
+    };
+
+    assert({
+      given: 'default action',
+      should: 'create correct actions',
```

Comment:
Try "should: return an action creator which returns an action with the correct type and payload"